tests/integration_tests/functional/test_cpu_features.py

```diff
@@ -239,7 +245,7 @@ def test_brand_string(test_microvm_with_api):
                    # Trigger Mode equal to Edge,
                    # and a Delivery Mode equal to Fixed.
                    # bit 0-7 represent interrupt vector that varies.
-]
+] + X2APIC_MSRS
```

Comment:
nit: which do you prefer?
```suggestion
    *[hex(i) for i in range(0x0000_0800, 0x0000_08FF)] # comment
]
```

Reply:
It is interesting but I personally find it more readable to avoid this syntax.